manage.py

```diff
@@ -0,0 +1,18 @@
+import os
+from flask_script import Manager, Shell
+from flask_migrate import Migrate, MigrateCommand
+from app import create_app, db
+from app import models
+
+app = create_app(os.getenv('APP_SETTINGS'))
+migrate = Migrate(app, db)
+manager = Manager(app)
+
+def make_shell_context():
```

Comment:
what does this function do?

Reply:
it adds objects to the import list to be to be registered with a `make_context` callback function which automatically imports  registered objects

saves the hectic of having to import the db instance and models every time a shell session is started when interacting with them from the terminal.

all you need to do is `python mange.py shell` and all you need is imported